### ISSUE TITLE
string literal `formData` breaks submissions collection

### ISSUE THREAD

**Issue by pmig** [excl-networks/strapi-plugin-ezforms#23]
Submitting a simple string as `formData` an error occurs. The data still gets stored inside the `ezforms_submission` table. But it is also not fetchable via the frontend. I needed to manually delete the effected submissions.

**Comment by Cpaczek** [comment]
Could you share the request your sending to ezforms/reproduction steps?

**Comment by Cpaczek** [comment]
Was unable to replicate with this post
```js
fetch("http://localhost:1337/api/ezforms/submit", {
  "headers": {
    "accept": "application/json, text/plain, */*",
    "accept-language": "en-US,en;q=0.9,ja;q=0.8",
    "cache-control": "no-cache",
    "content-type": "application/json",
    "pragma": "no-cache",
    "sec-ch-ua": "\"Not_A Brand\";v=\"99\", \"Google Chrome\";v=\"109\", \"Chromium\";v=\"109\"",
    "sec-ch-ua-mobile": "?0",
    "sec-ch-ua-platform": "\"Windows\"",
    "sec-fetch-dest": "empty",
    "sec-fetch-mode": "cors",
    "sec-fetch-site": "same-site"
  },
  "referrer": "http://localhost:3000/",
  "referrerPolicy": "strict-origin-when-cross-origin",
  "body": "{\"formName\":\"test\",\"formData\":\"`test`\"}",
  "method": "POST",
  "mode": "cors",
  "credentials": "omit"
});
```

**Comment by Cpaczek** [comment]
https://github.com/strapi/strapi/issues/15496 this is an issue with Strapi

**Comment by Cpaczek** [comment]
@pmig this will be fixed in 4.6.2 :)
